libcontainer/cgroups/systemd/common.go

```diff
@@ -11,6 +11,26 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+// Refer to https://github.com/coreos/go-systemd/blob/5a0db84d3dc459ccdc6ffcc44b1c452bf9f171cb/dbus/methods.go#L78-L101
+// or https://godoc.org/github.com/coreos/go-systemd/dbus#Conn.StartUnit
+// job completion: one of done, canceled, timeout, failed, dependency, skipped.
+// done indicates successful execution of a job. canceled indicates that a job
+// has been canceled  before it finished execution. timeout indicates that the
+// job timeout was reached. failed indicates that the job failed. dependency
+// indicates that a job this job has been depending on failed and the job hence
+// has been removed too. skipped indicates that a job was skipped because it
+// didn't apply to the units current state.
+type DbusJobCompletionResultType string
+
+const (
+	DbusJobDone       DbusJobCompletionResultType = "done"
+	DbusJobCanceled   DbusJobCompletionResultType = "canceled"
```

Comment:
Are you using any of those (I mean, except for `DbusJobDone`)?

Reply:
Remove now!